app/views/applications/process/summary.html.slim

```diff
@@ -0,0 +1,19 @@
+.row
```

Comment:
This template looks so good now, great! You can now easily understand what's being rendered here.